k2/csrc/log.h

```diff
@@ -283,6 +283,22 @@ inline bool DisableChecks() {
   return disable_checks;
 }
 
+/*
+   Get the max cpu memory (in bytes) can be allocated at a time through the
+   environment variable `K2_MAX_CPU_MEM_ALLOCATE`. Return the default value
+   (i.e. 200GB) if the variable does not set.
+ */
+inline int64_t MaxCpuMemAllocate() {
+  static std::once_flag init_flag;
+  // 200GB in bytes, 200 * 1024 * 1024 * 1024
+  static int64_t max_cpu_mem_allocate = 214748364800L;
+  std::call_once(init_flag, []() {
+    char *env_str = std::getenv("K2_MAX_CPU_MEM_ALLOCATE");
+    if (env_str != nullptr) max_cpu_mem_allocate = std::atol(env_str);
+  });
+  return max_cpu_mem_allocate;
+}
+
```

Comment:
We can also print it out in
https://github.com/k2-fsa/k2/blob/1b29f0a946f50186aaa82df46a59f492ade9692b/k2/python/k2/version/version.py#L60